radis/lbl/calc.py

```diff
@@ -531,6 +542,13 @@ def _check_molecules_are_consistent(
         # We add all of the DICT_INPUT_ARGUMENTS values:
         kwargs_molecule.update(**dict_arguments)
 
+        t = kwargs_molecule["mole_fraction"]
+        if flag:
+            if molecule not in {"CO2", "H2O", "He", "H2"}:
+                diluent["air"] -= t
```

Comment:
Depending on the database, these broadeners might or might not be available. For a given database, and depending on the molecule you're computing, broadeners might or might not be available too (i.e. it's possible that CO in HITRAN has CO2 broadening; but that N2 in HITRAN doesn't have CO2 broadneing).

Therefore:  no hardcoding is possible here. Better send all the correct diluent information; and let _calc_spectrum_one_molecule handle it (and use Air itself if not available)

Reply:
Yes, sir, I have changed it by checking in broadening functions whether the broadening coefficients are present are not, and in case they are not the present broadening coefficient of air is used.